.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,22 @@
+# Fast running checks for pull-requests
+
+name: Pull-Request Checks
+concurrency: 
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+on:
+  pull_request
+
+jobs:
+  check-freeze-period:
+    if: github.base_ref == 'master'
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/verifyFreezePeriod.yml@master
+  check-merge-commits:
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/checkMergeCommits.yml@master
```

Comment:
I'm not sure if this check is desired for JDT as here merge commits are used more regularly as far as I can tell. But all other JDT repositories have it enabled too. Therefore I assume it's fine.

Reply:
If other JDT repos use it it's fine.